doc/RecordingControl.xml

```diff
@@ -356,6 +358,18 @@ Change Request 2061, 2063, 2065, 2109</revremark>
           time intervals before and after the event may be captured.</para>
         <para>The recording job relies on the receiver service for receiving the data from other devices through receiver objects identified by ReceiverTokens</para>
       </section>
+      <section>
+        <title>External targets</title>
+        <para>The target interface allows configuration for devices that support recording to external storage targets. 
+          For authentication configuration see the related storage configuration APIs of the core specification.
+        </para>
+        <para>The target API defines for each recording a path on the storage device where
+          the related recordings should be stored. In order to keep reasonably sized files the
+          recordings are split into snippets. This specification defines a date and time based
```

Comment:
I did not see a definition of what a snippet is within the specification. Since the recording section already mentions files, should that be the terminology used here?

Reply:
Done.